src/server/debugcmd.h

```diff
@@ -12,15 +13,28 @@ class EngineShardSet;
 class ServerFamily;
 
 class DebugCmd {
+ private:
+  struct PopulateOptions {
+    uint64_t total_count = 0;
+    std::string_view prefix{"key"};
+    uint32_t val_size = 0;
+    bool populate_random_values = false;
+    struct SlotRange {
+      SlotId start = 0;
+      SlotId end = 0;
+    };
```

Comment:
You could use the existing `SlotRange` from `cluster_config.h`: https://shaharmike.com/dfly/dragonfly/src/server/cluster/cluster_config.h.html#dfly::ClusterConfig::SlotRange